tests/lib/authentication.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+from time import time
+import os
+import jwt
+import requests
+import json
+
+
+class Authentication():
+    @staticmethod
+    def generate_int_authentication():
+        with open("dev-private.key", "r") as f:

```

Comment:
this won't work on the CI - this private key should be available on an environmental variable - check with @ChewieCB